storage_service/storage_service/settings/base.py

```diff
@@ -29,6 +29,14 @@ def get_env_variable(var_name):
         raise ImproperlyConfigured(error_msg)
 
 
+def is_true(env_str):
+    return env_str.lower() in ["true", "yes", "on", "1"]
+
+
+def is_false(env_str):
+    return env_str.lower() in ["false", "no", "off", "0"]
```

Comment:
Rather than introducing another handler here, I'm just wondering if we need to change the names of the variables being used. 

According to the PR, the new settings will default to on. If we name the variables and update the descriptions to let users know they "disable" not "enable" we can still use: `is_true`. 

```
- **`SS_AUTH_PASSWORD_COMMON_VALIDATION`**:>> SS_AUTH_PASSWORD_COMMON_VALIDATION_OFF
- **`SS_AUTH_USER_ATTRIBUTE_SIMILARITY_VALIDATION`**: >> SS_AUTH_USER_ATTRIBUTE_SIMILARITY_VALIDATION_OFF
- **`SS_AUTH_PASSWORD_COMPLEXITY_VALIDATION`**: >> SS_AUTH_PASSWORD_COMPLEXITY_VALIDATION_OFF
```

Given it seems that we seem to only use these values to disable, I think that would be preferable to do. I think that also makes sense in-line as well where testing for `not false` is more complicated that truthiness/falseness.

Reply:
That aside, I was trying to work through some of the complexity in `base.py` and answer this question and wrote-up a sample wrapper for the `environ` function. For my money, `is_true` and our work casting and watching for `ValueErrors` inline is getting pretty heavy here. It's a clue for me that a different abstraction is needed (and I am sure must be used in other projects or modules available). I wonder if we'd be better served by a `get_envvar` wrapper for `environ.get`. Something like:

https://gist.github.com/ross-spencer/b30115cea336016e34897bb4e1d4db04

I don't do a lot of work in these areas, so it's interesting to come in cold with a microscope when doing CR. 

This is probably out of scope. But if you think it worth it that code should capture what I can see is most cases in this module.

Reply:
The proposed approach makes sense to me and I think it's good insight, but I'm inclined to agree that it's a bit out of scope. Especially given that we already have a [get_env_variable helper](https://github.com/artefactual/archivematica-storage-service/blob/stable/0.17.x/storage_service/storage_service/settings/base.py#L23-L29) with a slightly different focus that's in use in parts of `local.py` and `production.py` that are unrelated to this work but would need to be changed.

If we approach this in a separate piece of work, the [django-environ](https://github.com/joke2k/django-environ) library seems like something that might be worth looking into.